docs/rfcs/rfc-018-mathematical-operators.md

```diff
@@ -0,0 +1,126 @@
+# Mathematical operators
+
+## Problem
+As described in the issue #677, if we have to do a mathematical operation using the GraphQL library, 
+the common approach is to query the database and then perform the mutation with the final property value.
+The former approach has several downsides:
+- It requires multiple roundtrips to the database.
+- It's not concurrency safe.
+- It requires more code than necessary in the client.
+
+## Proposed Solution
+Add new input properties that solve atomic operations like:
+- `_ADD`
```

Comment:
Oh, it changes from `INC` to `ADD` since yesterday or not? I'm okay with both, but kind of prefer the shortest spelling/reading that is possible.

Reply:
Yes, I changed yesterday here 08a80e2c486611ccf062508ff14531fd4903eb64.
I like both long/short versions, Although I think that the DIV operator will be confused for an Integer Division operator with this name, that could be the case anyway. I think @darrellwarde prefer the long version, any takes on that @angrykoala ?

Reply:
I think the full words for these is appropriate. I think shortening is good if there is widely accepted convention for it (for example, `_LT`, `_GTE`), but the full words should be used otherwise (for example, `_CONTAINS`, `_STARTS_WITH`). I think these operators fall into the latter group.